Expanse v2 mirroring improvements (#14031)
demisto/content: Packs/ExpanseV2/ReleaseNotes/1_7_3.md

#### Integrations
##### Expanse v2

> - Added support for the **get-modified-remote-data** command which returns a list of modified incidents since the last update.

Reviewer: ```suggestion
- Added support for the **get-modified-remote-data** command which returns a list of modified incidents since the last update (used in the incident mirroring process from XSOAR version 6.1.0).
```

> - Added support for incidents reopening during the mirroring process.

Reviewer: ```suggestion
- You can now reopen incidents using the incident mirroring process.
```
